client/rpc/debug/src/lib.rs

```diff
@@ -712,7 +713,7 @@ where
 			let transactions = block.transactions;
 			if let Some(transaction) = transactions.get(index) {
 				let f = || -> RpcResult<_> {
-					let result = if trace_api_version >= 5 {
+					let result = if trace_api_version >= 7 {
```

Comment:
This concerns me. Did we skip version 6? How is it going from version `5` to `7`?